app/models/map-features/lot.js

```diff
@@ -412,10 +414,10 @@ export default class LotFragment extends MF.Fragment {
       return bldgclassLookup[this.bldgclass];
     }
 
-    @computed('cd')
-    get borocode() {
-      const borocd = this.cd;
-      return `${borocd}`.substring(0, 1);
+    @computed('borocode')
```

Comment:
(just commenting here what we discussed IRL) - you should able to just use the `borocode` attribute created above, instead of needing a computed property for this.